be/src/olap/tablet_manager.cpp

```diff
@@ -239,6 +239,15 @@ void TabletManager::cancel_unfinished_schema_change() {
             }
 
             AlterTabletTaskSharedPtr new_alter_task = new_tablet->alter_task();
+            if (new_alter_task != nullptr 
+                && (new_alter_task->related_tablet_id() != tablet->tablet_id() 
+                    || new_alter_task->related_schema_hash() != tablet->schema_hash())) {
+                LOG(WARNING) << "base tablet " << tablet->full_name()
+                             << "new tablet " << new_tablet->full_name()
```

Comment:
```suggestion
                             << " new tablet " << new_tablet->full_name()
```
add a space. The same the the following line.